### PR TITLE
punto7

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: .github/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Continues integration
```
Comment (holgiosalos) [inline .github/main.yml:1]: typo: Contin**ou**s

File: .github/main.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headed: true
+      - name: Cypress run
+        uses: cypress-io/github-action@v4
+        with:
+          build: npm run build
+          start: npm start
```
Comment (holgiosalos) [inline .github/main.yml:31]: EOF

File: .nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+node-version: 16.15.1
```
Comment (holgiosalos) [inline .nvmrc:1]: content should be something like: `v16.15.1`

File: package.json [hunk 4]
```diff
@@ -4,9 +4,9 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test:open": "npm run lint && cypress open",
-    "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    ... scripts existentes, no borrar
+    "test": "cypress run"
+    ... scripts existentes, no borrar
```
Comment (holgiosalos) [inline package.json:9]: it was required just to modify the test script

Comment (danielgalvis98) [inline package.json:9]: The change was not extactly this, maybe it is a little confussing but that where just comments to point out that the "scripts" section had already many scripts which shouldn''t be deleted.

The only change expected here is to add another script which is `"test": "cypress run"`, but let the other without changes

File: .github/main.yml (hunk 2 above)
Comment (holgiosalos) [inline .github/main.yml:31]: probably this is not required

Comment (danielgalvis98) [inline .github/main.yml:31]: This section is not needed, you are already running cypress tests on the previous step :)

Comment (jeffrey20122021) [reply .github/main.yml]: done all @holgiosalos

File: .nvmrc [hunk 5]
```diff
@@ -1 +1 @@
-node-version: 16.15.1
+node-version: v16.15.1
```
Comment (holgiosalos) [inline .nvmrc:1]: remove the `node-version:` part

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -24,8 +24,4 @@ jobs:
         with:
           browser: chrome
           headed: true
-      - name: Cypress run
-        uses: cypress-io/github-action@v4
-        with:
-          build: npm run build
-          start: npm start
\ No newline at end of file
+
```
Comment (holgiosalos) [inline .github/workflows/main.yml:27]: remove the whitespaces on line 27

File: .nvmrc (hunk 5 above)
Comment (jeffrey20122021) [reply .nvmrc:1]: done @holgiosalos

File: .github/workflows/main.yml (hunk 6 above)
Comment (jeffrey20122021) [reply .github/workflows/main.yml]: Done @holgiosalos

File: .nvmrc (hunk 5 above)
Comment (holgiosalos) [reply .nvmrc:1]: I still see it. The content of this file should ve only:
```
v16.15.1
```

Comment (jeffrey20122021) [reply .nvmrc]: done @holgiosalos

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**